LPP-48495

victorcdp opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

katienesterovich commented:
ci:test:sf

katienesterovich commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-portal-source-format/2832/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com//userContent/jobs/test-portal-source-format/builds/2832/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes 53 seconds 531 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/94f2d63a9302bd187ec887a8d39e2deffdb246c7">94f2d63a9302bd187ec887a8d39e2deffdb246c7</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-97294">LPS-97294</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/f63ba2eac2180172a78a1cc9ce27524a2c40a206">f63ba2eac2180172a78a1cc9ce27524a2c40a206</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-source-format/2832/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/2832/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest(master)/5338/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-19.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5338/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:x: ci:test:relevant - 26 out of 29 jobs passed in 2 hours 23 minutes 35 seconds 749 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7d5ad70a0a37afe498ace5df736928c41e551f0a">7d5ad70a0a37afe498ace5df736928c41e551f0a</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/a502cb923c272480139f9eb034d7265186c53522">a502cb923c272480139f9eb034d7265186c53522</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303926/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303923/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271979/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208258/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332593/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332594/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271982/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/354736/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342923/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 26 out of 29 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest(master)/5338/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/295824/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</strong></strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/292255/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</strong></strike></a></li></ul></div><details><summary><strong>26 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208257/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208260/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303926/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332595/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271981/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/354737/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271980/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303924/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208259/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303923/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271979/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208258/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332593/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/332594/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271977/">test-portal-acceptance-pullrequest-batch(master)/js-unit-private-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271978/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271976/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342924/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/271982/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208256/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/354736/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342922/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/354738/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342921/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303925/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342923/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><h5>For more details click <a href="https://test-1-19.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5338/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/295824/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/295824//consoleText">AXIS_VARIABLE=0,label_exp=!master #295824</a><ol><li><div><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/295824//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Forms_SubmitFormWithRequiredTextField_">LocalFile.Forms#SubmitFormWithRequiredTextField</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-19/1573151415770/test-portal-acceptance-pullrequest(master)/5338/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.Forms_SubmitFormWithRequiredTextField/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-19/1573151415770/test-portal-acceptance-pullrequest(master)/5338/functional-tomcat90-mysql57-jdk8/2/0/LocalFile.Forms_SubmitFormWithRequiredTextField/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-19/1573151415770/test-portal-acceptance-pullrequest(master)/5338/functional-tomcat90-mysql57-jdk8/2/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/292255/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/292255//consoleText">AXIS_VARIABLE=0,label_exp=!master #292255</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/292255//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Forms_AddFormInGlobal_">LocalFile.Forms#AddFormInGlobal</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-19/1573151415770/test-portal-acceptance-pullrequest(master)/5338/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.Forms_AddFormInGlobal/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-19/1573151415770/test-portal-acceptance-pullrequest(master)/5338/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.Forms_AddFormInGlobal/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-19/1573151415770/test-portal-acceptance-pullrequest(master)/5338/functional-tomcat90-mysql57-jdk8/3/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
These test failures look related. Can you see if they are not reproducible without your fix and reproducible with your fix?

brianikim commented:
See previous comment

alvarosaugarlr committed:
LPS-150388 Default admin to initialize dynamically with user input

alvarosaugarlr committed:
LPS-155999 Modify page title according to the context.

For LPS-150388.

alvarosaugarlr committed:
LPS-155999 Cancel button serves no purpose when initializing the default admin.

For LPS-150388.

alvarosaugarlr committed:
LPS-150388 Control user exception in Autologin class to clean the log

alvarosaugarlr committed:
LPS-156201 It is necessary to control the exception because in the feature LPS-150388, in the first access is done as admin and not as Guest which is the original behavior.

For LPS-150388.

LuisOrtiz89 committed:
LPS-158743 Marking the UpgradeReport class as an upgrade related class for the log context

LuisOrtiz89 committed:
LPS-158743 Printing the upgrade report in the console as thread context information

LuisOrtiz89 committed:
LPS-158743 Improving parseability of the upgrade report when printed as log context information

lfbesada committed:
LPS-149968 Providing a way to reindex an item without notifying the contributors.

JournalArticle versions are persisted in the same table, so they are the same base model. When an article is indexed we should reindex the other versions of the same article to keep updated the indexed documents. For example, when a new article version is published, depending on the configured value in "Index All Article Versions Enabled, the previous indexed version:
- If disabled: must be deleted.
- If enabled: must be updated because when publishing a new version the previous version will not be HEAD anymore.

If we don't provide a way to reindex without notifying the contributor, when we try to reindex a journal article from the contributor callback method will cause an infinity loop. That is because we can't determine when an article is reindexing because it has changed or because another version of the same article has changed.

lfbesada committed:
LPS-149968 Providing notification to the writer contributor when a document is deleted.

Otherwise, the contributors don't detect the cases that an index modification that happens as a consequence of the execution of a method with the annotation: @Indexable(type = IndexableType.DELETE)

lfbesada committed:
LPS-149968 Inline to prevent the modelDeleted notification when we are reindexing with IndexerWriterMode.DELETE.

LuisOrtiz89 committed:
LPS-158743 Adding filter to inject to make sure we are getting the right component

LuisOrtiz89 committed:
LPS-158743 Adding integration tests to check the upgrade log context information

LuisOrtiz89 committed:
LPS-158743 Fixing tests due to missing property setup

LuisOrtiz89 committed:
LPS-158743 Improving the UpgradeReport code to make it much more easy to add new sections to it in the future and have less code

LuisOrtiz89 committed:
LPS-158743 Changing tests to fit the new upgrade report format

rafaprax committed:
LPS-177178 Remove leftovers immediate=true from segments-asah-connector

achaparro committed:
LPS-158743 Omit Upgrade as first word for every piece of info in the Upgrade Report

achaparro committed:
LPS-158743 Simpler keys

achaparro committed:
LPS-158743 Do not remove period for properties

achaparro committed:
LPS-158743 Underline with the exact number of chars

achaparro committed:
LPS-158743 Manual SF

LuisOrtiz89 committed:
LPS-158743 Fixing tests due to changes in upgrade report format

LuisOrtiz89 committed:
LPS-158743 Improving upgrade report readability

LuisOrtiz89 committed:
LPS-158743 Fixing integration tests by adding one missing check and ignoring case in comparisons due to case insensitive databases

nkkwon committed:
LPS-175620 Add error message for fillObjectAction macro

nkkwon committed:
LPS-175620 Add path for error message

nkkwon committed:
LPS-175620 Add CannotSaveUpdateAnObjectEntryActions test

susanchen3 committed:
LPS-158743 Update upgrade report tests to fit new format

brianchandotcom committed:
LPS-158743 This is done already. Every test method has a new instance of the test class

brianchandotcom committed:
LPS-158743 Sort

brianchandotcom committed:
LPS-158743 Use SB

brianchandotcom committed:
LPS-158743 Sort? #sfme We need to update SF to check LinkedHashMapBuilder too

brianchandotcom committed:
LPS-158743 Wordsmith

brianchandotcom committed:
LPS-158743 Wordsmith

brianchandotcom committed:
LPS-158743 Better this way. Note that I added a . in the check

brianchandotcom committed:
LPS-158743 auto SF

brianchandotcom committed:
LPS-158743 Simplify

brianchandotcom committed:
LPS-158743 Inline

brianchandotcom committed:
LPS-158743 Proper name?

LuisOrtiz89 committed:
LPS-158743 Better naming

LuisOrtiz89 committed:
LPS-158743 Inline

LuisOrtiz89 committed:
LPS-158743 We cannot order alphabetically the keys on the LinkedHashMaps because we need the upgrade report to be printed in a specific order to improve readability

LuisOrtiz89 committed:
LPS-158743 More clear message

LuisOrtiz89 committed:
LPS-158743 Auto SF

Commit:
LPS-155999 buildLang.

For LPS-150388.

julschong committed:
LPS-178022 Remove Stream/Optional usages in SearchDisplayContext

julschong committed:
LPS-178022 Remove Stream/Optional usages in AssetCategoriesSearchFacetDisplayContextBuilder

julschong committed:
LPS-178022 Remove Stream/Optional usages in SearchLayoutFactoryImpl

julschong committed:
LPS-178022 Remove Stream/Optional usages in ModifiedFacetCalendarDisplayContextBuilderTest

julschong committed:
LPS-178022 Remove Optional usages in PortletPreferencesLookup

julschong committed:
LPS-178022 Apply to usages

julschong committed:
LPS-178022 SF, fix import

julschong committed:
LPS-178022 Remove Optional usages in PortletSharedRequestHelper

julschong committed:
LPS-178022 Apply to usages

julschong committed:
LPS-178022 Remove Stream/Optional usages in SearchResultSummaryDisplayContextBuilder

LuisOrtiz89 committed:
LPS-158743 UnsyncStringWriter should not be static, so every test has its own writer

achaparro committed:
LPS-158743 We do not use TableCounts class for this. Let's clarify by using map and a singular compound name. We are storing a map of table counts (all tables and one count per table)

achaparro committed:
LPS-158743 When needed

achaparro committed:
LPS-158743 Unify message

achaparro committed:
LPS-158743 Not needed

susanchen3 committed:
LPS-158743 Capitalize column names

julschong committed:
LPS-178022 Remove Optional usages in SearchInsightsPortlet

julschong committed:
LPS-178022 Remove Optional usages in SiteFacetPortlet

julschong committed:
LPS-178022 Remove Optional usages in RedirectSuggestionsMVCActionCommand

pyoo47 committed:
LRCI-3491 Move ci-forward logic from beanshell into java classes

Commit:
LPS-156497 SF Simplify

For LPS-150388.

julschong committed:
LPS-178022 Remove SearchArrayUtil, no more usage

julschong committed:
LPS-178022 Remove Stream usages from AssetEntriesSearchFacet

julschong committed:
LPS-178022 Remove Stream usages from FolderTitleLookupImpl

julschong committed:
LPS-178022 Remove Stream usages from FolderSearchFacetDisplayContextBuilder

julschong committed:
LPS-178022 Remove Stream usages from ScopeSearchFacetDisplayContextBuilder

julschong committed:
LPS-178022 Remove Stream usages from ModifiedFacetDisplayContextBuilder

julschong committed:
LPS-178022 Remove Stream usages from SuggestionsPortletDisplayContextBuilderTest

julschong committed:
LPS-178022 Remove Stream usages from SuggestionsPortletDisplayContextBuilder

julschong committed:
LPS-178022 Remove Stream usages from SuggestionsPortletDisplayContextBuilder

Commit:
LPS-150388 No exception is thrown if we use fetch instead of get

Commit:
LPS-150388 SF rename to make it clearer that it is for an initial setup

Commit:
LPS-150388 Because default admin is asserted based on repurposed user fields by portal-impl/**/UserLocalServiceImpl, all checks should be in portal-impl also for a cleaner separation of concern with modules

dantewang committed:
LPS-176210 Remove counter.jdbc.prefix as the separate counter data source has been removed

dantewang committed:
LPS-176210 Remove test usages

Commit:
LPS-177830 Create test to filter by related object fields using comparison operators through multiple relationships

Commit:
LPS-178464 Create test to filter by related object fields using logical operators through multiple relationships

Commit:
LPS-177834 Create test to filter by related object fields using string operators through multiple relationships

pyoo47 committed:
LRCI-3491 Don't notify when force push is detected.

dantewang committed:
LPS-178022 SF

dantewang committed:
LPS-178022 SF, move method to its only usage class

dantewang committed:
LPS-178022 Fix unit test

LuisOrtiz89 committed:
LPS-158743 Not needed and better naming

Commit:
LPS-150388 Fix regression which forces all users to change the reminder also

Commit:
LPS-150388 Apply the util to other code to deduplicate and centralize logic

andreluis2022 committed:
LPS-179056 Added automation test WithSpecialCharacters

joseabelenda committed:
LPS-179960 Move DynamicObjectDefinitionTable and DynamicObjectRelationshipMappingTable from service to api

joseabelenda committed:
LPS-179960 Sort

joseabelenda committed:
LPS-179960 Export com.liferay.object.dsl package

joseabelenda committed:
LPS-179960 Auto SF

dantewang committed:
LPS-178022 Fix wrong condition

BarbaraCabrera committed:
LPS-178510 Adapt the test to take into account the pid use in the LPS-177421

Co-authored-by: Víctor Galán <victor.galan@liferay.com>

lfbesada committed:
LPS-149968 Auto

lfbesada committed:
LPS-149968 Baseline

pyoo47 committed:
LRCI-3491 Move CI Forward logic into CIForwardProcessor

pyoo47 committed:
LRCI-3491 SF

pyoo47 committed:
LRCI-3491 Only the contructor and execute() should be public

pyoo47 committed:
LRCI-3491 SF

pyoo47 committed:
LRCI-3491 Temporarily restore PullRequest to original state to maintain compatibility during switch over. Revert this commit after switch over is complete.

Commit:
LPS-150388 SemVer

pyoo47 committed:
LRCI-3491 Clean up exception handling

pyoo47 committed:
LRCI-3491 Rename

Kvale1733 committed:
LRQA-79972 Add enviornments to service builder test suilte

pyoo47 committed:
LRCI-3491 Use new build property names

julschong committed:
LPS-177268 Remove Stream usages in FieldResourceTest

thektan committed:
LPS-163888 LPS-180128 portal-search-web: Fix clear button spacing in widget templates

achaparro committed:
LPS-158743 We are returning the value

achaparro committed:
LPS-158743 Be consistent with the table rows

achaparro committed:
LPS-158743 We are checking if the logContext contains a specific value

achaparro committed:
LPS-158743 String is singular

achaparro committed:
LPS-158743 Not var needed

achaparro committed:
LPS-158743 String is singular - inline

4lejandrito committed:
LPS-179564 Returning the total count in the page object

achaparro committed:
LPS-158743 Clearer naming

Commit:
LPS-180064 Fixed case results no more add another issues when edit

welitonpe committed:
LPS-180063 Fixed lost reference for Testray page to calculate filter position

welitonpe committed:
LPS-180067 Fixed missing column information

pyoo47 committed:
LRCI-3491 Add missing comments

kevenleone committed:
LPS-180066 Redirect to previous page when deleting last item from current page

pyoo47 committed:
LRCI-3491 Make it possible to override the retry message.

pyoo47 committed:
LRCI-3491 Leverage getRetryMessage() to post the retry comment as a side effect

kevenleone committed:
LPS-180066 SF

jgarcialr committed:
LPS-180189 set current company in thread local to be able to search for users instead on relying in default company

4lejandrito committed:
LPS-179564 Fixing NPE when page = 0

https://github.com/liferay/liferay-portal/blob/43b763b02f23e3ac3c1c8f25ba5c505ec157d0e4/modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/context/provider/PaginationContextProvider.java#L43

michaelhashimoto committed:
LRCI-3531 Remove deploy-workspace-client-extensions since it is no longer needed

michaelhashimoto committed:
LRCI-3531 Allow running poshi tasks against a given workspace

pyoo47 committed:
LRCI-3491 Close the pull request if forwarding was successful

michaelhashimoto committed:
LRCI-3531 Add scripts to setup the liferay sample workspace

abhnerramos committed:
LPS-180227 Add paths that were on ClientExtensionThemeSVG.testcase

michaelhashimoto committed:
LRCI-3496 Add one sample workspace poshi test

abhnerramos committed:
LPS-180227 Updating path on testcase that was being called directly

kevenleone committed:
LPS-180232 Create CompareRun Hyperlink

joseabelenda committed:
LPS-179960 Rename package

pyoo47 committed:
LRCI-3491 Ensure that the pull request was successfully forwarded before adding the success comment.

michaelhashimoto committed:
LRCI-3496 Add poshi-ext.properties to the .gitignore file

michaelhashimoto committed:
LRCI-3496 Ignore the poshi folder when updating all other workspaces

pyoo47 committed:
LRCI-3491 Move comment body generation logic to private methods

kevenleone committed:
LPS-180232 SF

michaelhashimoto committed:
LRCI-3531 Add one test suite that is able to run liferay-sample-workspace

michaelhashimoto committed:
LRCI-3431 Add workspace results into merge-test-results

kevenleone committed:
LPS-180232 Add missing breadcrumb step

Commit:
LPS-179975 Add Activities Component

Commit:
LPS-179975 Add Table List Component

Commit:
LPS-179975 Add Table List Mobile

Commit:
LPS-179975 Add Panel Component

Commit:
LPS-179975 Add Claim Activities Details

Commit:
LPS-179975 Refactoring Policy Activities Details

michaelhashimoto committed:
LRCI-3517 Save the poshi files for workspaces if available

Kvale1733 committed:
LPS-179403 Update Tomcat version to 9.0.73

pyoo47 committed:
LRCI-3491 Simplify

susanchen3 committed:
LPS-173364 Re-add lol properties

murilostodolni committed:
LPS-180352 Add validation to don't permit use expression builder when field is picklist as state

murilostodolni committed:
LPS-180352 Change conditional to return first

CalumR23 committed:
POSHI-511 Check if poshi script starts with "var"

Luizcarlosqueiroz committed:
LPS-178053 Set portalURL dynamically

Luizcarlosqueiroz committed:
LPS-178053 Fix argument value

pyoo47 committed:
LRCI-3491 Add message for stable failures

kenjiheigel committed:
POSHI-512 Use local variable

kenjiheigel committed:
POSHI-512 Check against simple classCommandName

kenjiheigel committed:
POSHI-512 No longer used

kenjiheigel committed:
POSHI-512 Use com.liferay.poshi.core:1.0.108

peerkar committed:
LPS-180275 portal-search: Fix language condition

lfbesada committed:
LPS-178510 SF rename

lfbesada committed:
LPS-178510 Assert the configuration properties instead the map we use to updated it

lfbesada committed:
LPS-178510 Prevent NPE if not initialized

lfbesada committed:
LPS-178510 SF extract constant and inline

lfbesada committed:
LPS-178510 SF extract to reuse

lfbesada committed:
LPS-178510 Add an assert to ensure the configuration hasn't changed

lfbesada committed:
LPS-178510 Add a new test to ensure the company configuration could be updated using different

lfbesada committed:
LPS-178510 SF use a local variable

lfbesada committed:
LPS-178510 SF rename

Commit:
LPS-180279 Modify macros to support portalURL

JoyceWang08211 committed:
LRQA-80042 Move portal configuration tests to Core Infra team

ambrinchaudhary committed:
LPS-179994 Set -- as status for KB Folders

victorg1991 committed:
LPS-180190 Don't allow tabs to be shrinked

brianchandotcom committed:
POSHI-512 prep next

brianchandotcom committed:
LPS-158743 SF

brianchandotcom committed:
LPS-158743 At least call it this

brianchandotcom committed:
LPS-158743 Rename for clarity

brianchandotcom committed:
LPS-158743 auto SF

davisaints committed:
LPS-180300 Change inappropriate paths

brianchandotcom committed:
LPS-158743 Rename

liulukiwi committed:
LPS-180183 Modify locator for web content repeatable field

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Inline

brianchandotcom committed:
LPS-158743 auto SF

davisaints committed:
LPS-180243 Set feature flag property for ticket related tests

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 auto SF

brianchandotcom committed:
LPS-158743 Move logic

brianchandotcom committed:
LPS-158743 Not needed, but better to always set vars in case we need it one day in filterMessages

brianchandotcom committed:
LPS-158743 Make things even more obvious

brianchandotcom committed:
LPS-158743 SF

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 auto SF

brianchandotcom committed:
LPS-158743 Inline

brianchandotcom committed:
LPS-158743 SF

brianchandotcom committed:
LPS-158743 The entry point for using this is generateReport which is NOT a static method

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Sort

brianchandotcom committed:
LPS-158743 SF

brianchandotcom committed:
LPS-158743 Use method

brianchandotcom committed:
LPS-158743 Rename

brianchandotcom committed:
LPS-158743 Rename/auto SF

brianchandotcom committed:
LPS-158743 Auto SF

Commit:
LPS-179975 Refectoring Claim Details Page

Commit:
LPS-179975 SF

brianchandotcom committed:
LPS-179960 Move it to the right module

naraandrade committed:
LPS-180194 Refactor GetAppModal to receive only the app id
- Make requests for delivery API instead of admin API.

naraandrade committed:
LPS-180194 liferay-npm-scripts lint:fix

naraandrade committed:
LPS-180194 liferay-npm-scripts format

brianchandotcom committed:
LPS-150388 SF

brianchandotcom committed:
LRCI-3496 SF

brianchandotcom committed:
LRCI-3496 Let me know if this works?

brianchandotcom committed:
LPS-177834 SF

brianchandotcom committed:
LRCI-3491 Wordsmith

brianchandotcom committed:
LRCI-3491 prep next

brianchandotcom committed:
LPS-177268 SF

gamerson committed:
LPS-178706 update Dockerfile

gamerson committed:
LPS-178706 more ignores

brianchandotcom committed:
LPS-178706 Comment it out

brianchandotcom committed:
LPS-178706 Apply

brianchandotcom committed:
LPS-178706 Use U70

brianchandotcom committed:
LPS-178510 SF

davisaints committed:
LPS-176849 Add CanFormulaFieldBeUsedWithEmailNotification test

jgarcialr committed:
LPS-180189 use setWithSafeCloseable method

brianchandotcom committed:
Revert "LPS-150388 SF"

This reverts commit 47be6d258c41bed504fc346105b926fc834dab06.

brianchandotcom committed:
LPS-149968 SF

kenjiheigel committed:
POSHI-512 These should not be final, the objects should always be recreated every test run

brianchandotcom committed:
LPS-178022 SF

joshchong committed:
LPS-179790 Use macro for save and continue

joshchong committed:
LPS-179790 Create and use PortletEntry.inputNumber macro for consistency

joshchong committed:
LPS-179790 Add reindex warning test

kenjiheigel committed:
POSHI-511 Use com.liferay.poshi.core:1.0.109

austinchiang committed:
LPS-180405 Add DM testray component to publications tests

brianchandotcom committed:
Baseline

brianchandotcom committed:
POSHI-511 SF

brianchandotcom committed:
POSHI-511 prep next

Commit:
LPP-48495 add default values for discontinued date

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
